src/components/NewLocationPage/CountyMap/CountyMap.tsx

```diff
@@ -61,6 +61,24 @@ const MAP_TYPE_INFO: { [key in MapType]: MapTypeInfo } = {
   },
 };
 
+function useMapTileOffset(): number {
+  const [yPosition, setYposition] = useState(0);
+  const pageHeight = document.documentElement.scrollHeight;
+  const footerHeight = document.getElementsByTagName('footer')[0]?.scrollHeight;
+  // Add an additional 500px to detect when the user is close enough to the bottom of the page.
+  const offsetPoint = pageHeight - footerHeight - 500;
+  // When the user goes beyond the offset point, vertically offset the map tile by the footer height + 100px.
+  const offsetAmount = yPosition > offsetPoint ? footerHeight + 100 : 0;
```

Comment:
If you rename the `offsetAmount` property on `MapContainer` rename this to match.